k8s/conf/skydive.yaml

```diff
@@ -54,7 +56,7 @@ data:
     ui:
       topology:
         favorites:
-          nsm-filter: "G.V().Has('Type', 'container', 'Docker.Labels.io.kubernetes.pod.namespace', 'default').In('Type', 'netns').Descendants().As('namespaces').G.V().Has('Type', 'host').As('hosts').Select('namespaces', 'hosts')"
+          nsm-filter: "G.V().Has('Type', 'container', 'Docker.Labels.io.kubernetes.pod.namespace', 'nsm-system').In('Type', 'netns').Descendants().As('namespaces').G.V().Has('Type', 'host').As('hosts').Select('namespaces', 'hosts')"
```

Comment:
Yeah, that is not great, but we don't have a better way to parametrise this. I guess @matrohon can have an idea of how to be more namespace flexible here. That should be a follow-up PR, though.